distribution/kernel/carbon-home/repository/resources/conf/templates/repository/conf/user-mgt.xml.j2

```diff
@@ -23,9 +23,9 @@
                 <UserName>{{super_admin.username}}</UserName>
                 <Password>{{super_admin.password}}</Password>
             </AdminUser>
-            <EveryOneRoleName>everyone</EveryOneRoleName>
+            <EveryOneRoleName>{{realm.every_one_role_name}}</EveryOneRoleName>
 
-            <OverrideUsernameClaimFromInternalUsername>true</OverrideUsernameClaimFromInternalUsername>
+            <OverrideUsernameClaimFromInternalUsername>{{realm.override_username_claim_from_internal_username}}</OverrideUsernameClaimFromInternalUsername>
```

Comment:
Are the users changing this config?

Reply:
This config has been moved from the carbon.xml and is mentioned in a previous migration doc. Can we omit?

Reply:
can you check it and see if this is really needed? also this key itself is not giving a clear idea on its functionality. Can you please check and come up with a better name if possible?

Reply:
The template for this config was removed